File: server/core/workingDay.go
package core

import "time"

// WorkingDay represents a single days in a month with working data.
type WorkingDay struct {
	Date      time.Time
	IsWorking bool
	Hours     int

	// TODO:PETERO review this
	UserID string `json:"user_id" bson:"userId"`
	RepoID string `json:"id" bson:"repoId"`
}

File: client/src/models/workingDay.ts
export default class WorkingDay {
  public date: Date;
  public hours: number;
  public groupId: string;

  constructor(date: Date, hours: number, groupId: string) {
    this.date = date;
    this.hours = hours;
    this.groupId = groupId;
  }
}
